src/Adapter/Module/Module.php

```diff
@@ -366,6 +368,30 @@ public function onReset(): bool
         return is_callable([$this->instance, 'reset']) ? $this->instance->reset() : true;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function onRegisterHook($hookName): bool
+    {
+        if (!$this->hasValidInstance()) {
+            return false;
+        }
+
+        return (bool) $this->instance->registerHook($hookName);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function onUnregisterHook($hookName): bool
```

Comment:
```suggestion
    public function onUnregisterHook(string $hookName): bool
```

Reply:
These methods must be added to the interface!